imaging_install.sh

```diff
@@ -148,6 +148,9 @@ else
     read -p "Cannot find the apache group name for your installation. Please provide? " group
 fi
 
+# Making lorisadmin part of the apache group
+sudo usermod -a -G $group $USER
+
```

Comment:
@christinerogers ,
this was already in your Readme but the installation script did not do it.